### ISSUE TITLE
May not work with Widget apps in most recent version of API?

### ISSUE THREAD

**Issue by HanClinto** [evonove/mkm-sdk#2]
I've been wrestling with this since last night, and I'm not sure if it's something that I'm doing wrong, or if it's a problem in the library.  I've confirmed that my application tokens work in other contexts (such as the [C# example here](https://www.mkmapi.eu/ws/documentation/API:Auth_csharp)).  However, when attempting to use mkm-sdk with my same tokens, I always get: "UnauthorizedAccess: Request failed. Status code: 401. Response message: Unauthorized."  

I don't know if this is because of recent security changes to OAuth, or simply because mkm-sdk doesn't function properly with Widget apps.  

The [MKM documentation](https://www.mkmapi.eu/ws/documentation/API:Auth_Overview) notes that widget apps are only supposed to use empty strings for both the Access Token and Access Secret.  Is that causing the problem with mkm-sdk, or is it something else?

Apologies if this isn't the correct place to post the issue -- I really wasn't sure.

This is running on Windows 7 -- all libraries and dependencies installed using Pip and confirmed as up-to-date.  

My usage example is:

```python
from mkmsdk.mkm import mkm 
import os

os.environ['MKM_APP_TOKEN'] = '<snip>'
os.environ['MKM_APP_SECRET'] = '<snip>'
os.environ['MKM_ACCESS_TOKEN'] = ''
os.environ['MKM_ACCESS_TOKEN_SECRET'] = ''

response = mkm.market_place.games()

print response
```

**Comment by Alien1993** [comment]
This SDK was created with Dedicated apps in mind and it was never tested with Widget apps, I'll look into it and release a new version in the future.

**Comment by HanClinto** [comment]
I'm more than happy to provide my test keys if you would like to use my widget app for testing.

Thank you so much for the SDK -- it is fantastic!

**Comment by Alien1993** [comment]
I'll let you know if I need anything, thank you.

**Comment by Alien1993** [comment]
Fixed in PR #3, it will be released with the next milestone in less than a week. 

@HanClinto sorry it took so long. :)
